resources/lib/ui/control.py

```diff
@@ -65,6 +66,21 @@ def xbmc_add_dir(name, url, iconimage='', description=''):
     liz=xbmcgui.ListItem(name, iconImage="DefaultFolder.png", thumbnailImage=iconimage)
     liz.setInfo('video', infoLabels={ "Title": name, "Plot": description })
     liz.setProperty("fanart_image", iconimage)
+    sysaddon = sys.argv[0]
```

Comment:
we should add those as extra actions only if logged in,
also, this should be done via parameter from `default.py` call